internal/mode/static/nginx/modules/test/httpmatches.test.js

```diff
@@ -403,7 +403,7 @@ describe('redirect', () => {
     method: 'GET',
     headers: ['header1:value1', 'header2:value2'],
     params: ['Arg1=value1', 'arg2=value2=SOME=other=value'],
-    redirectPath: '/a-match',
+    redirectPath: '/a-match?Arg1=value1&arg2=value2%3DSOME%3Dother%3Dvalue',
```

Comment:
let's remove `?Arg1=value1&arg2=value2%3DSOME%3Dother%3Dvalue'` from the redirectPath. Note that because we removed the following block, this means a returned redirect path cannot include them:
```
  // If performing a rewrite, $request_uri won't be used,
  // so we have to preserve args in the internal redirect.
  let args = qs.stringify(r.args);
  if (args) {
    args = '?' + args;
  }

  r.internalRedirect(match.redirectPath + args);
```
Note that we also need to  update the corresponding test case `redirects to the redirectPath of the first match the request satisfies'` and remove `?Arg1=value1&arg2=value2%3DSOME%3Dother%3Dvalue'` from the expected path there.